pkg/kf/options.go

```diff
@@ -36,6 +36,10 @@ type pushConfig struct {
 	Output io.Writer
 	// ServiceAccount is the service account to authenticate with
 	ServiceAccount string
+	// MinScale is the lower scale bound
+	MinScale int
+	// MaxScale os the upper scale bound
+	MaxScale int
```

Comment:
This file is auto-generated, you should be able to put these in options.yml and run `go generate ./...` to auto-generate these fields and the options.

The only caveat is that the generator is very basic, so it won't have the `WithPushScaleBounds(min, max int)`, but you can copy that into `push.go` if you like.